core/modules/punch-ability/client/punch-ability.js

```diff
@@ -1,8 +1,15 @@
 import audioManager from '../../../client/audio-manager';
 
+const getRandomFloat = (min, max, decimals) =>
```

Comment:
Is it possible to move this function to the core "misc" file? I think it will be useful elsewhere 🙂 (there is "randomInRange" which exists in this file but for integers)